src/lib/log/middleware.go

```diff
@@ -1,17 +1,18 @@
 package log
 
 import (
-	"log"
 	"net/http"
 )
 
-// Log ... リクエストログ掃き出し用のmiddleware
-func Logger(h http.Handler) http.Handler {
+// Middleware ... output request log
+func Middleware(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		rAddr := r.RemoteAddr
 		method := r.Method
 		path := r.URL.Path
-		log.Printf("Remote: %s [%s] %s", rAddr, method, path)
-		h.ServeHTTP(w, r)
+		ctx := r.Context()
+		ctx = NewLogger(ctx)
```

Comment:
It is not good that the creation of Logger is hidden.
Should not be given an instance of Logger as an argument?
ex:
```go
func Middleware(next http.Handler, logger *zap.Logger) http.Handler {
}
```